src/configCreator.ts

```diff
@@ -70,24 +68,20 @@ async function createOptions(
       //   example: a rule for the storybook should only apply to files with
       //            "story" or "stories" in the name. If enabled for all files it
       //            reports false positives on normal files.
-      //            check: conf file @ eslint-plugin-storybook/configs/recomneded.js
+      //            check: conf file @ eslint-plugin-storybook/configs/recommended.js
       const [storybookPatterns, otherPatterns] = partition(patterns, (p) =>
         p.patternId.startsWith("storybook")
       )
 
-      debug(
-        `[codacy]: do we have plugins to apply only to some file types? - ${
-          storybookPatterns.length > 0
-        }`
-      )
+      debugWhen(storybookPatterns.length > 0, "[DEBUG] we have plugins to apply only to some file extensions")
 
       const result = cloneDeep(defaultOptions)
       if (result.baseConfig) {
         // remove extends and overrides from our default config.
-        result.baseConfig.extends = []
-        result.baseConfig.overrides?.forEach(
-          (override: any) => (override.extends = [])
-        )
+        //result.baseConfig.extends = []
```

Comment:
Should this be commented here? Was this throwing an error?

Reply:
By being there It was ignoring the added plugin recommend config/rules in **eslintDefaultOptions.ts** and so preventing some rules to be properly interpreted.
It's up for debate though. Do you think it should not be ignored?